lib/SIL/Parser/ParseSIL.cpp

```diff
@@ -8621,13 +8621,22 @@ bool SILParserState::parseSILCoverageMap(Parser &P) {
         break;
       }
 
-      auto Counter = State.parseSILCoverageExpr(Builder);
-      if (!Counter) {
-        BodyHasError = true;
-        break;
+      using MappedRegion = SILCoverageMap::MappedRegion;
+
+      if (P.Tok.is(tok::identifier) && P.Tok.getText().equals("skipped")) {
```

Comment:
I think checking for contextual keyword would be better here. Not sure if SIL has escaped identifiers but this will check that we didn’t write backticks around `skipped`.

```suggestion
      if (P.Tok.isContextualKeyword("skipped")) {
```

Reply:
Ah nice